tests/foreman/ui_airgun/test_reporttemplates.py

```diff
@@ -386,6 +161,7 @@ def test_positive_e2e_crud(session, module_org, module_loc):
         'input_content.description': gen_string('alpha')
     }]
     with session:
+        # C
```

Comment:
you kidding, right?

Reply:
No. "C" for first part of CRUD. I can expand it to the full form if you think that really matters.

Reply:
Yeah, please expand with 'create xxx (whatever you create)'. Man, please don't think that I want to offend your feelings, but we are working here on serious commercial product and that doesn't sound acceptable